retriever/lib/repository.py

```diff
@@ -69,9 +70,14 @@ def check_for_updates():
                 try:
                     os.remove(os.path.normpath(os.path.join(HOME_DIR, "scripts", script_name)))
                     _download_from_repository("scripts/" + script_name,
-                                              os.path.normpath(os.path.join(SCRIPT_WRITE_PATH, script_name)))
+                                              os.path.normpath(os.path.join(SCRIPT_WRITE_PATH, script_name)),
+                                              repo)
                 except Exception as e:
                     print(e)
                     pass
     except:
         raise
+
+    if repo == RETRIEVER_REPOSITORY:
```

Comment:
Can you run me through your logic for checking if the repo is a scripts repo vs just the normal retriever repo?

Reply:
@zhangcandrew in the method `check_for_updates`, it is done from both the repositories, retriever (for the five legacy scripts) and recipes (for the rest of the scripts). I just used the above  logic to call the same method with different repositories (to update from both of them).

Reply:
Ok I see, so this is a placeholder for now, got it!